### ISSUE TITLE
feature request: able to hide/show mouse on demand from command line

### ISSUE THREAD

**Issue by mailinglists35** [jcs/xbanish#26]

**Comment by grahamgower** [comment]
`xbanish -a` / `pkill xbanish` takes care of this, no?

**Comment by Amund211** [comment]
I've implemented this in [my fork](https://github.com/Amund211/xbanish/commit/5ba1a25b55eb12c0af888894f3a03c1581e41161). It uses signals, so you can do `pkill -USR1 xbanish` to hide the cursor and `pkill -USR2 xbanish` to show it. Do note that it is quite hacky, and just barely works for my purposes.

**Comment by 4eUeP** [comment]
@Amund211 Thank you for your idea about using signals. However, there may a bug in this implementation. Run `pkill -USR1 xbanish` will call `hide_cursor`, but the next mouse moving event will call `show_cursor` again. So I need set `always_hide = 1` while receiving the `SIGUSR1` signal. 

code: https://github.com/4eUeP/xbanish/commit/ee6ea17033513f4c40095965d54c1ec50a10461d

**Comment by Amund211** [comment]
@4eUeP Yeah, that probably makes more sense. I only use the signal to show the cursor, so I hadn't though of that
